sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -275,22 +276,22 @@ export class JsonRpcProvider extends Provider {
   ): Promise<GetTxnDigestsResponse> {
```

Comment:
```suggestion
  cursor = null,
  limit = null
  ordering = descending,
  ): Promise<GetTxnDigestsResponse> {
```

Since we already introduced a breaking change in the response type, I think we should adopt the new interface as well